modules/nextMillenniumBidAdapter.js

```diff
@@ -1,17 +1,37 @@
-import { isStr, _each, parseUrl, getWindowTop, getBidIdParameter } from '../src/utils.js';
-import { getRefererInfo } from '../src/refererDetection.js';
+import {
+  _each,
+  createTrackPixelHtml,
+  deepAccess,
+  isStr,
+  getWindowTop,
+  getBidIdParameter,
+  logMessage,
+  parseUrl,
+  triggerPixel,
+} from '../src/utils.js';
+
+import CONSTANTS from '../src/constants.json'
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import * as events from '../src/events.js'
+
 import { registerBidder } from '../src/adapters/bidderFactory.js';
-import { BANNER } from '../src/mediaTypes.js';
+import { getRefererInfo } from '../src/refererDetection.js';
 
 const BIDDER_CODE = 'nextMillennium';
 const ENDPOINT = 'https://pbs.nextmillmedia.com/openrtb2/auction';
 const TEST_ENDPOINT = 'https://test.pbs.nextmillmedia.com/openrtb2/auction';
 const SYNC_ENDPOINT = 'https://statics.nextmillmedia.com/load-cookie.html?v=4';
 const TIME_TO_LIVE = 360;
 
+const EXPIRENCE_WURL = 20 * 60000
+const wurlMap = {}
+
+events.on(CONSTANTS.EVENTS.BID_WON, bidWonHandler)
+cleanWurl()
+
 export const spec = {
   code: BIDDER_CODE,
-  supportedMediaTypes: [BANNER],
+  supportedMediaTypes: [BANNER, VIDEO],
```

Comment:
This property isn't currently being used

Reply:
https://docs.prebid.org/dev-docs/bidder-adaptor.html
According the doc from the prebid.org is property exists as example.